packages/react-mutation-mapper/src/store/DefaultMutationMapperStore.ts

```diff
@@ -162,6 +165,11 @@ class DefaultMutationMapperStore implements MutationMapperStore {
         );
     }
 
+    @computed
+    public get myCancerGenomeData(): IMyCancerGenomeData {
+        return fetchMyCancerGenomeData();
```

Comment:
I don't think the function should be called `fetch` if its not actually doing any fetching, a little confusing IMO

Reply:
Right, it's not fetching anything, my cancer genome data is all static. I kept the original function name but probably it's better to rename it.